src/peergos/shared/user/UserContext.java

```diff
@@ -118,6 +119,179 @@ public TransactionService getTransactionService() {
         );
     }
 
+    @JsMethod
+    public App.Todo getTodoApp() {
```

Comment:
All this being built in to Peergos concerns me. A large part of the point of the todo list is to dog food how we expect others to build apps that are isolated from our code. That means minimal changes to things like UserContext, and certainly nothing this app-specific.

Reply:
This is version 1. I would like to see the above in a separate PR.

Reply:
Fine to do it this way first, but I don't think we should merge or deploy it like this.